### DISCUSSION TITLE
Training API - EDS-NLP

### DISCUSSION THREAD

**Discussion by giscus** [aphp/edsnlp#342] [Announcements]
# Training API - EDS-NLP

https://aphp.github.io/edsnlp/latest/tutorials/training/

**Comment by percevalw** [comment]
Typo (todo fix): because the transformer is actually stored in nlp.pipes.span_classifier.embedding.embedding (span_classifier → pooler → transformer), the first group in optimizer.groups should not be `^transformer` but `span_classifier[.]embedding[.]embedding`
